pytorch/inference/docker/1.6.0/py3/Dockerfile.gpu

```diff
@@ -0,0 +1,114 @@
+FROM nvidia/cuda:10.1-cudnn7-devel-ubuntu16.04
+# NCCL_VERSION=2.4.7, CUDNN_VERSION=7.6.2.24
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="2"
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+# Add arguments to achieve the version, python and url
+ARG PYTHON_VERSION=3.6.10
+ARG TS_VERSION=0.1.1
+
+# See http://bugs.python.org/issue19846
+ENV LANG C.UTF-8
+ENV LD_LIBRARY_PATH /opt/conda/lib/:$LD_LIBRARY_PATH
+ENV PATH /opt/conda/bin:$PATH
+ENV SAGEMAKER_SERVING_MODULE sagemaker_pytorch_serving_container.serving:main
+ENV TEMP=/home/model-server/tmp
+
```

Comment:
Similarly, the GPU binary URL will be:

```
ARG PT_INFERENCE_URL=https://aws-pytorch-binaries.s3-us-west-2.amazonaws.com/r1.6.0_inference/20200727-223446/b0251e7e070e57f34ee08ac59ab4710081b41918/gpu/torch-1.6.0-cp36-cp36m-manylinux1_x86_64.whl
```

Reply:
Do you have a similar URL for Torchvision ?

Reply:
Yes, for gpu it is:
```
https://torchvision-build.s3.amazonaws.com/1.6.0/cpu/torchvision-0.7.0-cp36-cp36m-manylinux1_x86_64.whl
```

Reply:
This URL is not accesible. Just confirming

Reply:
addressed offline :)

Reply:
Done